src/subdomains/supporting/bank/special-external-iban/special-external-iban.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+import { SpecialExternalIban, SpecialExternalIbanType } from './special-external-iban.entity';
+import { SpecialExternalIbanRepository } from './special-external-iban.repository';
+
+@Injectable()
+export class SpecialExternalIbanService {
+  constructor(private readonly specialExternalIbanRepo: SpecialExternalIbanRepository) {}
+
+  async getMultiAccountIban(): Promise<SpecialExternalIban[]> {
+    return this.specialExternalIbanRepo.findBy({ type: SpecialExternalIbanType.MULTI_ACCOUNT_IBAN });
+  }
+
+  async getBlacklist(): Promise<SpecialExternalIban[]> {
+    return this.specialExternalIbanRepo.findBy([
+      { type: SpecialExternalIbanType.BANNED_IBAN },
+      { type: SpecialExternalIbanType.BANNED_BIC },
+    ]);
```

Comment:
Could use `{ type: In([SpecialExternalIbanType.BANNED_IBAN, SpecialExternalIbanType.BANNED_BIC]) },`